app/src/main/java/com/isc/hermes/controller/PopUp/TextInputPopup.java

```diff
@@ -70,16 +70,25 @@ private void setOnClickListener() {
      * This method validates the input text from the TextInputPopup.
      */
     private void validateInput() {
-        String text = inputText.getText().toString();
-        if (TextUtils.isEmpty(text)) {
+        String text = inputText.getText().toString().trim();
+        if (TextUtils.isEmpty(text) || validateSpaceInput(text)) {
             setErrorMessage("This field is required");
-        } else if (text.length() > 20) {
+        } else if (text.length() > 15) {
```

Comment:
Validation should be use a constant configurable instead a hardcoding number for prevent future changes or requirements of validation

Reply:
The requested change has been made